core/src/main/scala/org/apache/spark/rdd/ReliableCheckpointRDD.scala

```diff
@@ -83,8 +84,9 @@ private[spark] class ReliableCheckpointRDD[T: ClassTag](
    * Return the locations of the checkpoint file associated with the given partition.
    */
   protected override def getPreferredLocations(split: Partition): Seq[String] = {
-    val status = fs.getFileStatus(
-      new Path(checkpointPath, ReliableCheckpointRDD.checkpointFileName(split.index)))
+    val path = new Path(checkpointPath, ReliableCheckpointRDD.checkpointFileName(split.index))
```

Comment:
Do we expect to be able to recover check-pointed RDDs across Spark versions? My impression that isn't something that is really supported.